codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/customization/ApiGatewayExportsNullabilityExceptionIntegration.java

```
@@ -0,0 +1,191 @@
+            ShapeId.from("com.amazonaws.route53recoverycontrolconfig#Route53RecoveryControlConfig"),
+            ShapeId.from("com.amazonaws.pinpoint#Pinpoint"),
+            ShapeId.from("com.amazonaws.apigatewayv2#ApiGatewayV2"),
+            ShapeId.from("com.amazonaws.mediaconvert#MediaConvert"),
+            ShapeId.from("com.amazonaws.medialive#MediaLive"),
+            ShapeId.from("com.amazonaws.macie2#Macie2"),
+            ShapeId.from("com.amazonaws.mediapackage#MediaPackage"),
+            ShapeId.from("com.amazonaws.apigatewaymanagementapi#ApiGatewayManagementApi"),
+            ShapeId.from("com.amazonaws.kafka#Kafka"),
+            ShapeId.from("com.amazonaws.mediapackagevod#MediaPackageVod"),
+            ShapeId.from("com.amazonaws.mq#mq"),
+            ShapeId.from("com.amazonaws.iot1clickdevicesservice#IoT1ClickDevicesService"),
+            ShapeId.from("com.amazonaws.serverlessapplicationrepository#ServerlessApplicationRepository"),
+            ShapeId.from("com.amazonaws.schemas#schemas"),
+            ShapeId.from("com.amazonaws.pinpointsmsvoice#PinpointSMSVoice"),
+            ShapeId.from("com.amazonaws.route53recoveryreadiness#Route53RecoveryReadiness"),
+            // APIGW services migrated to Smithy
+            ShapeId.from("com.amazonaws.dataexchange#DataExchange"),
+            ShapeId.from("com.amazonaws.kafkaconnect#KafkaConnect"),
+            ShapeId.from("com.amazonaws.mediatailor#MediaTailor"));
+
+    @Override
+    public Model preprocessModel(Model model, GoSettings settings) {
+        ShapeId service = settings.getService();
+        if (!APIGW_NULLABILITY_SERVICES.contains(service)) {
+            return model;
+        }
+        try {
+            return handleApiGateWayExportsNullabilityExceptions(model, service);
+        } catch (Exception e) {
+            LOGGER.severe("Failed to handle APIGW exports nullability exceptions for service: " + service.toString());
```

Any reason to ignore the error?